modules/admin_manual/pages/configuration/files/file_versioning.adoc

```diff
@@ -72,11 +72,11 @@ The versions are adjusted along this pattern every time a new version is created
 
 WARNING: The versions app never uses more that 50% of the user’s storage quota. If the stored versions exceed this limit, ownCloud deletes the oldest file versions until it meets the disk space limit again.
 
-TIP: Adjust the `'versions_retention_obligation'` setting in `config.php` to avoid filling up the user's quota.
+TIP: Adjust the `'versions_retention_obligation'` setting in `config.php` to avoid filling up the user's quota. For details see the chapter below.
```

Comment:
```suggestion
TIP: Adjust the `'versions_retention_obligation'` setting in `config.php` to avoid filling up the user's quota. For details see the section below.
```